ui/helpers/utils/util.js

```diff
@@ -689,12 +689,13 @@ export const checkTokenIdExists = (address, tokenId, obj) => {
     // Convert to decimal
     convertedTokenId = hexToDecimal(tokenId);
   }
-
-  if (obj[address]) {
-    const value = obj[address];
+  // Convert the input address to checksum address
+  const checkSumAdr = toChecksumHexAddress(address);
```

Comment:
Both addresses `0x80acdde51b1f379ae60f72203bfd24d4d6d4412d `and `0x80AcDDE51B1f379AE60f72203bfD24D4d6D4412d`
Are valid Hex addresses (the fct `isValidHexAddress `would return true for both addresses)

But only `0x80AcDDE51B1f379AE60f72203bfD24D4d6D4412d  `is a valid checksum address. (the fct isValidChecksumAddress would return true for this one and false for the other)

The user's nft object is saved with a checksum address. This is why we are converting the input address to a checksum address.